payslip_line_manually/models/hr_payslip.py

```diff
@@ -0,0 +1,56 @@
+from odoo import fields, models
+
+
+class HrPayslip(models.Model):
+    _inherit = "hr.payslip"
+
+    line_manually_ids = fields.One2many(
+        "hr.payslip.line.manually",
+        "res_id",
+        string="Manual Payslip Lines",
+        domain=[("model", "=", "hr.payslip")],
+        context={"default_model": "hr.payslip"},
+        copy=True,
+    )
+
+    def _compute_payslip_line(self, rule, localdict, lines_dict):
```

Comment:
I think you must call super()._compute_payslip_line here, because if you don't, the module will not be compatible with other inheriting this function.